test/unit-tests/controller.test.ts

```diff
@@ -314,10 +320,17 @@ contract(
 
         const vaultCounterAfter = new BigNumber(await controllerProxy.getAccountVaultCounter(accountOwner1))
         assert.equal(vaultCounterAfter.minus(vaultCounterBefore).toString(), '1', 'vault counter after mismatch')
+        assert.equal(finalMarginPool, marginPool.address, 'vault margin pool mismatch')
       })
 
       it('should open vault from account operator', async () => {
+        await borrowableMarginPool.setOptionsVaultWhitelistedStatus(accountOwner1, true, { from: owner })
```

Comment:
I don't like how in this test pretty much all of the accountOwner1 tests are now only done on the borrowable margin vault.  I'd prefer that we do on two different owners one that is borrowable and one that is regular and do tests on each.  Could even just create a 2nd unit test which repeats all the code but just whitelists the one account in one case and doesn't in 2nd.